app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -74,4 +76,10 @@ def set_access_control_headers
     headers['Access-Control-Allow-Origin'] = '*'
   end
 
+  def ab_test_onboarding(community_id:)
```

Comment:
I think this name should be more verb-y, because it has side-effects? Like `assign_something_test`